src/applications/disability-benefits/all-claims/utils/submit.js

```diff
@@ -752,6 +752,52 @@ export const addFileAttachments = formData => {
   return { ...clonedData, ...(attachments.length && { attachments }) };
 };
 
+export const addToxicExposure = formData => {
+  // If no toxic exposure data exists, return unchanged
+  if (!formData.toxicExposure) {
+    return formData;
+  }
+
+  // If user selected only "none" for conditions, remove all toxic exposure data except conditions
```

Comment:
Is this an additional failsafe if the destructive action modal does not remove toxic exposure keys?

Reply:
Move this logic into another PR